typing/typeclass.ml

```diff
@@ -2148,8 +2148,10 @@ let report_error env ppf = function
         "@[The instance variable is %s;@ it cannot be redefined as %s@]"
         mut1 mut2
   | No_overriding (_, "") ->
-      fprintf ppf "@[This inheritance does not override any method@ %s@]"
-        "instance variable"
+      fprintf ppf
+        "@[This inheritance does not override any methods@ \
+         or instance variables@ but is explicitly marked as@ \
+         overriding with `!`.@]"
```

Comment:
The definition right below this uses 
```
`%s'
```

whereas you refer to the exclamation mark with two backticks. Which one's right?

Reply:
My opinion is that neither are right: I have an upcoming PR that introduces a `inline_code` tag and uses that tag uniformly across all error messages.
I tend to think that this the best way to enforce an uniform quoting style.

Reply:
That does sound much better!